war/.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Normally despite the shell script, it's supported on Windows (https://typicode.github.io/husky/#/?id=does-it-work-on-windows)
Now, it could be interesting to get some Windows users to ensure it's really working.
On my own system it was working, but perhaps I have too many stuff installed that are necessary for Jenkins development.